cli/tests/integration_tests.rs

```diff
@@ -45,6 +45,35 @@ fn std_lint() {
   assert!(status.success());
 }
 
+#[test]
+fn doc_tests() {
+  for file in util::root_path()
+    .join("cli/tests/doc")
+    .as_path()
+    .read_dir()
+    .unwrap()
+  {
+    let file = file.unwrap();
+    assert!(file.file_type().unwrap().is_file());
+
```

Comment:
@SyrupThinker there's really no need to add integration tests for this PR. You could add new test case in `cli/doc/tests.rs`:

```rust
 assert!(
    colors::strip_ansi_codes(super::printer::format(entries).as_str())
      .contains("Generic<[string, number]>")
```